management/dbaas/xtra_db_cluster_test.go

```diff
@@ -97,11 +97,25 @@ func TestXtraDBClusterServer(t *testing.T) {
 			assert.True(t, foundPXC, "Cannot find PXC with name %s in cluster list", name)
 		}
 
+		showXtraDBClusterParamsParam := xtra_db_cluster.ShowXtraDBClusterParams{
+			Context: pmmapitests.Context,
+			Body: xtra_db_cluster.ShowXtraDBClusterBody{
+				KubernetesClusterName: kubernetesClusterName,
+				Name:                  "first-pxc-test",
+			},
+		}
+		xtraDBCluster, err := dbaasClient.Default.XtraDBCluster.ShowXtraDBCluster(&showXtraDBClusterParamsParam)
+		assert.NoError(t, err)
+		assert.Equal(t, xtraDBCluster.Payload.Name, "first-pxc-test")
+		assert.Equal(t, xtraDBCluster.Payload.Username, "root")
```

Comment:
Should it only return user and no password?

Reply:
Password is hardcoded for now - so I don't see the big benefit of testing it.